sdcm/provision/common/provisioner.py

```diff
@@ -8,26 +8,13 @@
 TagsType = Dict[str, str]
 
 
-class ProvisionParamsBase(BaseModel):  # pylint: disable=too-few-public-methods
-    name: str
+class ProvisionParameters(BaseModel):  # pylint: disable=too-few-public-methods
+    name: str  # Name of the parameter needed only to make distinction between the parameter instances
     region_name: str
     availability_zone: str
-    duration: float = None
-    price: float = None
-
-
-class ProvisionGuaranteedParams(ProvisionParamsBase):  # pylint: disable=too-few-public-methods
-    """
-    Provision request parameters for instances that COULD NOT BE REVOKED by cloud provider
-    Price for such instances are usually higher
-    """
-
-
-class ProvisionNotGuaranteedParams(ProvisionParamsBase):  # pylint: disable=too-few-public-methods
-    """
-    Provision request parameters for instances that COULD BE REVOKED by cloud provider
-    Price for such instances are usually lower
-    """
+    revocable: bool  # Signals on weather instance could be revoked by cloud provider
```

Comment:
- is it spot?
- weather -> whether

Reply:
In aws terminology revocable+price = spot

Reply:
per our conversation rename revocable -> spot

Reply:
done